forum/qa-theme/SnowFlat/qa-styles.css

```diff
@@ -4595,24 +4595,42 @@ pre[class*="brush:"]
 	overflow: scroll;
 }
 
-.syntaxhighlighter-parent,
 .syntaxhighlighter
 {
-	position: relative;
+	max-height: var(--code-block-raw-height); /* provided by inline style from JavaScript */

```

Comment:
chciałbym, żebyś wydzielił ten zmieniany kod css do osobnego pliku wedle własnego uznania. Zróbmy z tym porządek tak, aby jasno oddzielić kod q2a od naszych zmian w css. Myślę, że jest to realne do zrobienia. Jeśli nie chcesz, to za możesz za karę spróbować zrobić update wersji forum do najnowszej ;)

Reply:
Mogę to wydzielić, ale zrobiłbym to w osobnym PR, dedykowanym wydzieleniu wszystkich customowych CSSów do osobnego pliku.